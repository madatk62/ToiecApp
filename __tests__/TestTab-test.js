import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import TestTab from '../App/Components/TestTab';

test('renders correctly', () => {
    const tree = renderer.create(<TestTab darkMode={false} />).toJSON();
    expect(tree).toMatchSnapshot();
});