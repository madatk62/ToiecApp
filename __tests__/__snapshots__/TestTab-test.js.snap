// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#EEEEEE",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1976D2",
        "flexDirection": "row",
        "justifyContent": "center",
        "paddingBottom": 20,
        "paddingTop": 40,
      }
    }
  >
    <Text
      style={
        Object {
          "alignContent": "center",
          "color": "white",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      KIá»‚M TRA
    </Text>
  </View>
  <RCTScrollView
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</RCTSafeAreaView>
`;
